app/controllers/houses_controller.rb

```diff
@@ -0,0 +1,34 @@
+class HousesController < ApplicationController
+  def index
+    @house = House.all
+    render json: @house
```

Comment:
- [ ] It would be cleaner if you use `@houses` instead of `@house` here. Also, your `@house` variable could be set inside of a private function which you could call `before_action` for specific actions.

Reply:
done